.github/workflows/inactive_issue_checker.yml

```
@@ -0,0 +1,39 @@
+name: Check Inactive Issues
+permissions: read-all
+on:
```

This seems an odd trigger. Shouldn't it be a daily cron?

Perhaps coordinate with @Mayank77maruti since he is looking into implementing GitHub Actions to replace oppiabot; there should be a proper system for determining when each operation gets triggered.

---

Yeah once he had suggested me to divide jobs on the basis of their permissions will fix it now. 
@Mayank77maruti is anything else left you can suggest ?

---

It's not fixed. Please request a re-review only when it is.

---

@Mayank77maruti would it be like this, if we had to run workflow each mid-night. Or would it be nice to run this workflow at 3 days of interval, what say @seanlip ? 
```suggestion
on:
schedule:
    - cron: '0 0 * * *'
  # GitHub doesn't provide assurance that the scheduled jobs will run on time
  # (see https://github.community/t/no-assurance-on-scheduled-jobs/133753).
  # So, we add the workflow_dispatch event here to allow triggering this
  # workflow manually if needed.
  workflow_dispatch:
```

---

@Mayank77maruti could you PTAL on this schedule once, I hadn't fix it even.

---

@Mayank77maruti Please respond, thanks. This blocks the review/merge.

---

@Ashu463 Running the workflow each - midnight is better.

---

FIne @Mayank77maruti will update schedule as shown above. 
Thanks.